Shared/R4/files/BasicPatientInfoPrepopulation-0.2.0.cql

```diff
@@ -35,3 +35,13 @@ define CoverageResource: singleton from (
     where ('Coverage/' + coverage.id) = RequestCoverage[0].reference.value)
 
 define MedicareId: "CoverageResource".subscriberId.value
+
+// Get Patient home address
+define HomeAddress: singleton from (Patient.address address where address.use.value = 'home')
+define Line: HomeAddress.line[0].value
+define City: HomeAddress.city.value
+define State: HomeAddress.state.value
+define Zip: HomeAddress.postalCode.value
+
+define Telecom: Coalesce (Patient.telecom telecom where telecom.system.value = 'phone' )
```

Comment:
Why do we need Coalesce here?

Reply:
I think I was testing a patient/practitioner without a phone?  Let me retest and take it out if it's not needed.

Reply:
Actually, it's the opposite.  It's so that when there are more than one phone listed, it shows the first one.  I'm going to keep the Coalesce in there for now.  A better way is to print out all the phones listed in rank order, but that will require that we separate each out into multiple edit boxes.